grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJobExecutionListener.groovy

```diff
@@ -121,10 +124,16 @@ class ClientBatchJobExecutionListener implements JobExecutionListener {
 
         final String encodedContentAfterDate = URLEncoder.encode(contentAfterDate, 'utf-8')
         final String encodedPath = URLEncoder.encode(path, 'utf-8')
-        final grabPath = "http://${host}:${port}/grabbit/job?path=${encodedPath}&after=${encodedContentAfterDate}"
+
+        URIBuilder uriBuilder = new URIBuilder(scheme: "http", host: host, port: port as Integer, path: "/grabbit/job")
+        uriBuilder.addParameter("path", encodedPath)
+        uriBuilder.addParameter("after", encodedContentAfterDate)
+        for(String excludePath : excludePaths) {
+            uriBuilder.addParameter("excludePath", URLEncoder.encode(excludePath, 'UTF-8'))
+        }
```

Comment:
Yup that's what we decided to go with, the underlying structure takes away the issue of having the unique delimiter being the bottleneck of the communication between the client and the server... we would still need the delimiter for the job parameter though and changes to that would be minimal incase we need to change the delimiter(doesn't affect the communication between client and server)